beeline/middleware/django/__init__.py

```diff
@@ -158,6 +169,17 @@ def __call__(self, request):
                 for connection in connections.all():
                     stack.enter_context(connection.execute_wrapper(db_wrapper))
                 response = self.create_http_event(request)
+
+                def wrap_streaming_content(content):
```

Comment:
Could you elaborate on what the wrapping is achieving here? I think I can guess, but wanted to make sure I understood after our whole conversation in Slack. 🙂

Reply:
Yeah, this deserves a comment explaining it more -- the first `wrap_streaming_content`'s utility is more evident, but this one isn't.

The complexity is that `HoneyDBWrapper` only works for database queries run inside its `with contextlib.ExitStack()`.  Without this, the queries are run after the `response` is returned, well after the `contextlib.ExitStack()` has cleaned up the contexts that do the tracing.

So this `wrap_streaming_content` sets up a second context that runs later, in the context of when the generator is being called, that provides that same context again.  The first hunk above makes the request not end until the streaming is done; this one ensures that DB queries that happen while in that streaming section are recorded properly.